CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contribute
+
+CARTO VL platform is an open-source library. We are more than happy to receive your contributions to the code and the documentation as well.
+
+## Filling a ticket
+
+If you want to open a new issue in our repository, please follow these instructions:
+
+- Add a descriptive title.
+- Write a good description, answering the following questions:
+    - what is happening?
+    - what should happen?
+    - how to reproduce the issue. Remember to include Browser, OS and carto-vl version.
+- Try to add an example showing the problem (using JSFiddle, JSBin or a just code snippet).
+
+
+## Contributing code
+
+Best part of open source, collaborate in CARTO VL code! We like hearing from you, so if you have any bug fixed, or a new feature ready to be merged. Those are the steps you should follow:
+
+- Sign our Contributor License Agreement (CLA) [Learn more here](https://carto.com/contributions/)
+- Fork the carto-vl repository.
+- Create a new branch in your forked repository.
+- Commit your changes. Remeber to add new tests if possible.
+- Open a pull request.
+- Any of the carto-vl mantainers will take a look.
+- If everything works, it will merged and released \o/.
+- If you want more detailed information, [this GitHub guide](https://opensource.guide/how-to-contribute/) is a must.
+
+## Completing documentation
+
+CARTO VL documentation is located in docs/. That folder is the content that appears in the [Developer Center](https://carto.com/developers/carto-vl/support/contribute/). Just follow the instructions described in contributing code and after accepting your pull request, we will make it appear online :).
```

Comment:
Change "Just follow the instructions described in contributing code and after accepting your pull request, we will make it appear online :)." to "Follow the instructions described above and once we accept your pull request it will appear online :)."